examples/inductive_inference/inductive_inference_gridworld.ipynb

```diff
@@ -11,17 +11,26 @@
    "cell_type": "code",
    "execution_count": 1,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/home/conor/Documents/ml_at_verses/pymdp_dev/pymdp/pymdp/agent.py:706: SyntaxWarning: assertion is always true, perhaps remove parentheses?\n",
+      "  assert (\n"
+     ]
+    }
+   ],
```

Comment:
nit: could remove these extraneous outputs (can be a follow up PR).

although an issue to address the warning in pymdp/pymdp/agent.py:706` probably makes sense?